netbox/netbox/navigation/menu.py

```diff
@@ -290,6 +290,7 @@
                 get_model_item('extras', 'customfield', _('Custom Fields')),
                 get_model_item('extras', 'customlink', _('Custom Links')),
                 get_model_item('extras', 'exporttemplate', _('Export Templates')),
+                get_model_item('extras', 'imageattachment', _('Image Attachments'), actions=()),
```

Comment:
I'm not sure it makes sense to put this under customizations, though I also can't think of anywhere more fitting. It should probably come at the end of the list though.